src/app/order-list/order-list.component.css

```
@@ -0,0 +1,50 @@
+.order-category{
+    color: #3c1b00;
+    font-weight: bold;
+  }
+
+  .flex-container {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+  }
+
+  .column{
+    flex-direction: column;
+  }
+
+  .order-container{
+    margin: 50px 0;
+  }
+  
+  @keyframes zoom {
+    from {transform:scale(0)}
+    to {transform:scale(1)}
+  }
+  
+  .add-to-cart-btn{
+    background-color: #3c1b00;
+    color: #ffffff;
+  }
+  
+  
+  @media (max-width: 600px){
+    .order-container{
+      margin: 0;
+    }
+  }
+
+  .wrapper{
+    max-width: 800px;
+    margin: 0 auto;
+  }
+
+  .order-items{
+    flex:none;
+    margin: 5% 0;
+  }
+  
+  .order-price{
+    font-size:200%;
+    float:right;
```

Kad već koristiš flex, možda bi bolje bilo i ovdje ga iskorisititi-
```suggestion
    display: flex;
    justify-content: flex-end;
```

---

U redu :)